lib/executor/ramping_arrival_rate.go

```diff
@@ -471,3 +467,49 @@ func (varr RampingArrivalRate) Run(parentCtx context.Context, out chan<- stats.S
 	}
 	return nil
 }
+
+// runningVUs controls the activeVUs
+// executing the received requests for iterations.
+type runningVUs struct {
+	Iterations chan struct{}
```

Comment:
I :heart:  the approach of splitting this into a separate small component, though I'm not 100% sure about the names... Though my ideas aren't necessarily any better... :sweat_smile: `VUPool`? `ActiveVUPool`? 

In any case, `Iterations` probably should be not be exported and the comment looks a bit wonky.

Reply:
`activeVUPool` sounds slightly better to me.

Also `TryRunIteration` or `RunIterationMaybe` instead of `Run`. Not sure about the `Try`/`Maybe`, but `Iteration` should be part of it.